pkg/agent/flowexporter/connections/connections.go

```diff
@@ -190,16 +192,22 @@ func (cs *ConnectionStore) Poll() (int, error) {
 	// We do not expect any error as resetConn is not returning any error
 	cs.ForAllConnectionsDo(resetConn)
 
-	filteredConns, err := cs.connDumper.DumpFlows(openflow.CtZone)
+	filteredConnsList, totalConns, err := cs.connDumper.DumpFlows(openflow.CtZone)
 	if err != nil {
 		return 0, err
 	}
 	// Update only the Connection store. IPFIX records are generated based on Connection store.
-	for _, conn := range filteredConns {
+	for _, conn := range filteredConnsList {
 		cs.addOrUpdateConn(conn)
 	}
-	connsLen := len(filteredConns)
-	filteredConns = nil
+	connsLen := len(filteredConnsList)
+	filteredConnsList = nil
```

Comment:
not introduced by this PR, but why this line is needed?

Reply:
Yes, not required. It was a stray line from a past debugging price. Thanks for pointing it out. Should be removed in the future.